### PR TITLE
Dev to alpha

### PR DESCRIPTION
**Pull request by szuecs**
* remove limits for master components
* flannel as daemonset
* fix zmon crash on start with initContainers

### CODE REVIEW

File: cluster/manifests/flannel/daemonset.yaml [hunk 1]
```diff
@@ -66,15 +66,8 @@ spec:
...
-        nodeAffinity:
-          requiredDuringSchedulingIgnoredDuringExecution:
-            nodeSelectorTerms:
-            - matchExpressions:
-              - key: flannel
-                operator: NotIn
-                values:
-                - local
+      - key: node-role.kubernetes.io/master
+        effect: NoSchedule
```
Comment (linki) [inline cluster/manifests/flannel/daemonset.yaml:70]: I believe this toleration must be applied before rolling the nodes.

Comment (linki) [reply cluster/manifests/flannel/daemonset.yaml:70]: No, scratch that. Normally yes, but the Pod also has a "tolerate everything" toleration above. This is therefore not doing anything and was just added to be similar to the other DaemonSets.

**Comment by linki** [comment]
👍

**Comment by aermakov-zalando** [comment]
👍

**Pull request merged**